README.md

```diff
@@ -59,9 +59,21 @@ for the actor and a few configuration files.
 cd ./my/awesome/project
 apify init
 ```
-
 This command will only set up local actor development environment in an existing directory,
-i.e. it will create the `apify.json` file and `apify_local` directory.
+i.e. it will create the `apify.json` file and `apify_storage` directory.
+
+Before you can start to run your project using `apify run`, you have to set up right start command in `package.json` under scripts.start:
```

Comment:
Before you can run your project using `apify run`, you have to set up the right start command in `package.json` under `scripts.start`. For example: